packages/firestore/test/unit/local/persistence_test_helpers.ts

```diff
@@ -51,12 +51,17 @@ const LOCAL_STORAGE_PREFIX = 'firestore_';
  * any previous contents if they existed.
  */
 export async function testIndexedDbPersistence(
-  synchronizeTabs?: boolean
+  options: {
+    dontPurgeData?: boolean;
```

Comment:
What would you think of making this 'purgeData' and having it default to true if not provided?

Reply:
a) I don't like negative argument names
b) I don't like optional arguments that default to 'true'.

I slightly preferred compromising `a` over `b` in this case.